src/main/java/wolox/training/services/impl/BookServiceImpl.java

```diff
@@ -66,11 +68,20 @@ public Book createByOpenLibrary(String isbn) {
         }
     }
 
+    @Override
+    public List<Book> findByAllFields(Model model) {
```

Comment:
Model is very generic you could create a model request more representative

Reply:
Create a BookParametersDTO for all fields parameters